specutils/manipulation/model_replace.py

```diff
@@ -64,7 +64,8 @@ def model_replace(spectrum, replace_region, model=10, extrapolation_treatment='d
     >>> flux_val = np.array([2, 4, 6, 8, 10, 12, 14, 16, 18, 20])
     >>> input_spectrum = Spectrum1D(spectral_axis=wave_val * u.AA, flux=flux_val * u.mJy)
     >>> spline_knots = [3.5, 4.7, 6.8, 7.1] * u.AA
-    >>> result = model_replace(input_spectrum, None, spline_knots) # doctest: +IGNORE_OUTPUT
+    >>> model_replace(input_spectrum, None, spline_knots)
+    <Spectrum1D(flux=<Quantity [ 2.,  4.,  6.,  8., 10., 12., 14., 16., 18., 20.] mJy>, spectral_axis=<SpectralAxis [ 1.,  2.,  3.,  4.,  5.,  6.,  7.,  8.,  9., 10.] Angstrom>)>
```

Comment:
I dunno what this example is supposed to return. I don't have anything to refer to but the output is same as input, is that correct?

Reply:
The input spectrum is just a straight line, so I think a spline fit should just return a straight line as well. Makes sense to me, although it isn't a particularly enlightening example.

Reply:
Thanks for the clarification!

I think making examples enlightening is out of scope here. 😆